packages/fether-react/src/BackupAccount/BackupAccount.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React, { Component } from 'react';
+import { AccountHeader, Card, Form as FetherForm } from 'fether-ui';
+import { inject, observer } from 'mobx-react';
+import { accountsInfo$ } from '@parity/light.js';
+import light from '@parity/light.js-react';
+import { Link, withRouter } from 'react-router-dom';
+
+import withAccount from '../utils/withAccount';
+
+@withRouter
+@withAccount
+@light({
+  accountsInfo: accountsInfo$
+})
+@inject('createAccountStore')
+@observer
+class BackupAccount extends Component {
+  state = {
+    isLoading: false,
+    password: '',
+    message: ''
+  };
+
+  toggleMsg = msg => {
+    this.setState({
+      message: msg
+    });
+  };
+
+  handlePasswordChange = ({ target: { value } }) => {
+    this.setState({ password: value });
+  };
+
+  handleSubmit = event => {
+    const { accountAddress, createAccountStore, history } = this.props;
+    const { password } = this.state;
+
+    event.preventDefault();
+    this.setState({ isLoading: true });
+
+    // Save to parity
+    createAccountStore
+      .backupAccount(accountAddress, password)
+      .then(res => {
+        if (res) {
```

Comment:
What happens `if (!res)`?

Reply:
This should never happen since res would come from`return Promise.resolve('Successfully backed up account');`. Can think of either throwing so it's handled in the catch block or removing the `if(res)` block entirely.